lib/Common/Memory/RecyclerWriteBarrierManager.cpp

```diff
@@ -225,11 +231,6 @@ X64WriteBarrierCardTableManager::Initialize()
 
         LPVOID cardTableSpace = ::VirtualAlloc(NULL, _cardTableNumEntries, MEM_RESERVE, PAGE_READWRITE);
 
-        if (cardTableSpace == nullptr)
```

Comment:
Is this removed intentionally? cardTableSpace can't be nullptr here, or is not used?